vach/src/writer/leaf.rs

```diff
@@ -197,11 +187,14 @@ impl<'a> Default for Leaf<'a> {
 			handle: Box::<&[u8]>::new(&[]),
 			flags: Flags::empty(),
 			content_version: 0,
-			#[cfg(feature = "compression")]
-			compress: CompressMode::Never,
+
+			#[cfg(feature = "crypto")]
 			encrypt: false,
+			#[cfg(feature = "crypto")]
 			sign: false,
 
+			#[cfg(feature = "compression")]
+			compress: CompressMode::Never,
```

Comment:
Same here, wouldn't it make sense to, by default, compress?

Reply:
On it